<ui version="4.0" >
 <class>ConvertParamClass</class>
 <widget class="QDialog" name="ConvertParamClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Output Parameters</string>
  </property>
  <layout class="QFormLayout" name="formLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Frequency:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Bit per sample:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="mFreqCmb" />
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="mBpsCmb" />
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="mButtonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="mChannelCmb" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Channel Count</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
